docs/sdk/src/guides/async_backing_guide.rs

```diff
@@ -174,7 +174,7 @@
 //!    - In the `para_client` field, pass in a cloned para client rather than the original
 //!    - Add a `para_backend` parameter after `para_client`, passing in our para backend
 //!    - Provide a `code_hash_provider` closure like that shown below
-//!    - Increase `authoring_duration` from 500 milliseconds to 1500
+//!    - Increase `authoring_duration` from 500 milliseconds to 2000
```

Comment:
I would make it clear how this value is related to the configuration on the relay chain.